permissions/jolla-messages.profile

```diff
@@ -11,7 +11,6 @@
 ### APPLICATION
 # FIXME: This is a legacy D-Bus service name, we need to drop it at some point
 dbus-user.own org.nemomobile.qmlmessages
-# FIXME: another systematic dbus-name to deal with?
-dbus-user.own org.freedesktop.Telepathy.Client.qmlmessages
+dbus-user.own org.freedesktop.Telepathy.Client.${OrganizationName}.${ApplicationName}
```

Comment:
This is the rule we want but I'm not sure this is the place for it. Should we put this to Messages.permission?

Reply:
Well, should we? Maybe we should as this is for all messaging apps to use? I'll make the change.

Reply:
Moved with c1ca493